OPCB/functions/retrieve_help1.sqf

```diff
@@ -1,53 +1,53 @@
-	_civ = _this select 0;
-	_theFugitive = _this select 3 select 0;
+_civ = _this select 0;
+_theFugitive = _this select 3 select 0;
 
-	_questionedFugitive = _civ getVariable "isQuestionedFugitive";
-	_isFugitive = _civ getVariable "isFugitive";
+_questionedFugitive = _civ getVariable "isQuestionedFugitive";
+_isFugitive = _civ getVariable "isFugitive";
 
-	if(_isFugitive == "yes") then
-	{
-		hint "YOU WON'T GET ME ALIVE";
-		_enemySide = createGroup resistance;
-		removeAllWeapons _civ;
-		[_civ] joinSilent grpNull;
-		[_civ] joinSilent _enemySide;
+if(_isFugitive == "yes") then
+{
+	hint "YOU WON'T GET ME ALIVE";
+	_enemySide = createGroup resistance;
+	removeAllWeapons _civ;
+	[_civ] joinSilent grpNull;
+	[_civ] joinSilent _enemySide;
 
-		_civ addVest "V_TacVest_blk_POLICE";
-		_civ addWeapon "hgun_ACPC2_F";
-		_civ addMagazineCargo ["9Rnd_45ACP_Mag",1];
-		_civ enableAI "MOVE";
-		_civ action ['SwitchWeapon', _civ, _civ, 0];
-		_enemySide setCombatMode "COMBAT";
-	}
-	else
+	_civ addVest "V_TacVest_blk_POLICE";
+	_civ addWeapon "hgun_ACPC2_F";
+	_civ addMagazineCargo ["9Rnd_45ACP_Mag",1];
+	_civ enableAI "MOVE";
+	_civ action ['SwitchWeapon', _civ, _civ, 0];
+	_enemySide setCombatMode "COMBAT";
```

Comment:
I think you should also `reveal` all players near him and then have him force shoot at the nearest player for maximum effectiveness. >:D

Also add a `setUnitPos "UP"` right in there so he doesn't decide to go prone like an idiot when you switch to "COMBAT".